session-check.gemspec

```diff
@@ -8,13 +10,19 @@ Gem::Specification.new do |s|
   s.version = Session::Check::VERSION
   s.authors = ['Harry Lascelles']
   s.email = ['harry@harrylascelles.com']
-  s.homepage = 'https://github.com/firstbanco/session-check'
+  s.homepage = 'https://github.com/bambooengineering/session-check'
   s.summary = 'A gem for JS clients to check if their session has expired.'
   s.license = 'MIT'
 
   s.files = Dir["{app,lib,config}/**/*"] + ["README.md"]
   s.require_paths = ['lib']
 
-  s.add_dependency 'rails', '>= 5.0'
+  s.add_dependency 'rails', '>= 6.0'
```

Comment:
Does it matter that this gem is actually using rails 7? (AFAICT from the GEmfile.lock)

Reply:
I guess not, no. Will change back!